app/api.py

```diff
@@ -109,7 +113,10 @@ async def create_mentee(data: Mentee):
     Returns:
         New record data or schema discrepancy error as dictionary
     """
-    return {"result": API.db.create("Mentees", data.dict())}
+    try:
+        return {"result": API.db.create("Mentees", data.dict())} 
+    except DuplicateKeyError:
+        raise HTTPException(status_code=409, detail="Profile ID must be unique.")
 
```

Comment:
Elegant, lean coding. An improvement from the prior version.

Reply:
Hey thanks Ben